bundles/org.eclipse.ui.forms/src/org/eclipse/ui/forms/FormColors.java

```diff
@@ -535,7 +535,7 @@ else if (testTwoPrimaryColors(listSelection, 120, 256)
 		// LIST_SELECTION foreground colour
 		// Examples: Win High Con Black, Win High Con #1, Win High Con #2
 		// (covered in the second part of the OR clause above)
-		createColor(IFormColors.TITLE, rgb);
+		createColor(IFormColors.TITLE, getSystemColor(SWT.COLOR_TITLE_FOREGROUND));
```

Comment:
Even nicer and more consistent with the OS systems would IMHO be to use a gray color as background, not sure if we have a good constant for that.

Reply:
The palette is visible running SWT's ControlExample in the "Colors" tab. There are the COLOR_WIDGET_*_SHADOW to serve this. I personally don't mind about which exact color we pick, as long as it's readable, it's a system color and gradients are off by default. So please give a try and suggest a color and I can update.